src/charm.py

```diff
@@ -628,6 +632,17 @@ def _on_anonymize_user_action(self, event: ActionEvent) -> None:
                     f"Failed to anonymize user with username {username}:{ex.stdout}"  # type: ignore
                 )
 
+    def _stop_service(self):
+        """Stop discourse, this operation is idempotent."""
+        logger.info("Stopping discourse")
```

Comment:
Not a very big deal but I want to know if this logging line is intentional? Maybe we can add a bit more context to it?

Reply:
I found it interesting to put in the log that the service was stopped. As for the context, I'm not sure what could be added. This function is called just after a change of the charm's status that says why the charm is stopped

Reply:
Ah okay, thanks for clearing that up. It's ok for me then!